### DISCUSSION TITLE
Using condition in Front Matter

### DISCUSSION THREAD

**Discussion by rvmourik** [maizzle/framework#592] [Q&A]
Hi There,

I have a question:

I want two build environments, so i've setup the following two:

* Production (which is used as a online preview)
* Platform (which will be building "twig"-files for re-use in a Symfony Project)

What i want to do is something like this:

```
title: "{{ page.env === 'platform' ? '@@@{{ 'render_this_escaped' }}' : 'Render a preview text' }}"
```

**Output production environment**
```
<h1>Render a preview text</1>
```

**Output platform environment**
```
<h1>{{ 'render_this_escaped' }}</h1>
```

Hope someone can help me out with this, i want to prevent the use of two variables (one for platform, and one for production).

Thanks in advance.

Robbert

**Comment by husseinalhammad** [comment]
I don't use Maizzle myself, but generally speaking in YAML (front matter) you can nest values:

```yaml
title:
  dev: some value
  prod: another value
```

So perhaps instead of writing the condition in front matter you can write it in the template and render the appropriate value based on the environment.
